app/controllers/api/v1/parks_controller.rb

```diff
@@ -12,13 +12,7 @@ def show
     def create
         park = Park.new(name: params["name"], address: params["address"], city: params["city"], state: params["state"], zip: params["zip"], description: params["description"])
         if park.save
-            #flash[:notice] = "Park added successfully!"
-            # format.html { redirect_to parks_path, :controller => 'parks',  notice: 'Assignmenttable was successfully created.' }
             render json: { park: park, error: [], works: "Submitted Successfully!"}
-            # react router push for redirect
-            #render :action => "new"
-            #render {action: "index", controller: "parks"}
-
         else
             render json: { park: {}, error: park.errors.full_messages, works: "" }
```

Comment:
what exactly is `works` doing here? If it is useful, perhaps update it. Maybe `message` or `status`